src/confirm.rs

```diff
@@ -136,11 +136,11 @@ impl<'a> Confirm<'a> {
         let mut out = Buffer::ansi();
 
         out.set_color(&self.theme.title)?;
-        writeln!(out, " {}", self.title)?;
```

Comment:
[IMO]
This style of writing is still available, 
```rust
writeln!(out, " {}", XXXX)?;
```
but personally, I think this one seems better. What do you think?
```rust
write!(out, " ")?;
writeln!(out, "{}", XXXX)?;
```
The reason is that although the number of lines of code will increase, it's because "{}" is visually less noticeable. Alternatively, creating a common function like write_with_heading_whitespace seems like a good idea, but it might feel a bit redundant.